src/platform/utilities/sso/keepAliveSSO.js

```diff
@@ -40,3 +40,5 @@ export default async function keepAlive() {
     return {};
   }
 }
+
+export { keepAlive };
```

Comment:
Does this change make the `keepAlive` function both a `default` and named export?

Reply:
@billfienberg correct, I needed the named export so I could mock the calls in a few tests cases in src/platform/utilities/tests/sso.unit.spec.js